docs/website/developers/committers/release-process/publish.md

```diff
@@ -124,6 +150,8 @@ Tag the release in git
 Make a signed tag for this release, based on the tag for the release candidate, and then push the tag:
 
 {% highlight bash %}
+git checkout ${VERSION_NAME}
```

Comment:
This line should be unnecessary, as the `git tag` command on the next line explicitly places the tag on the commit `apache-brooklyn-${VERSION_NAME}-rc${RC_NUMBER}`